modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedSimpleFieldsStructure.testcase

```diff
@@ -477,7 +477,15 @@ definition {
 
 		DataEngine.duplicate(fieldFieldLabel = "Texto");
 
-		WebContentStructures.saveCP();
+		WaitForElementPresent(
+			key_text = "Save",
+			locator1 = "Button#ANY");
+
```

Comment:
When duplicating a Field, the save button is disabled quickly so the click does not find the button.
I added a `WaitforElementPresent` because of this delay.